eclipse-scout-core/src/widget/Widget.ts

```diff
@@ -2364,16 +2388,11 @@ export class Widget extends PropertyEventEmitter implements WidgetModel, ObjectW
    *          an optional options object. Shorthand version: If a string is passed instead
    *          of an object, the value is automatically converted to the option {@link ScrollToOptions.align}.
```

Comment:
Improve description of shorthand? While it is technically still a string, it should now only be one defined by the type ScrollToAlignment.